gokart/mypy.py

```diff
@@ -361,65 +360,115 @@ def _collect_parameter_args(self, expr: Expression) -> tuple[bool, dict[str, Exp
             return True, args
         return False, {}
 
-    def _infer_task_on_kart_attr_init_type(self, sym: SymbolTableNode, context: Context) -> Type | None:
```

Comment:
(just a memo for this change)
This function aims to infer type for pydantic field, which is not needed on gokart.